src/ad-hoc-visualizations/issues/visualization.tsx

```diff
@@ -56,10 +56,26 @@ export const IssuesAdHocVisualization: VisualizationConfiguration = {
         }),
     getIdentifier: () => AdHocTestkeys.Issues,
     visualizationInstanceProcessor: () => VisualizationInstanceProcessor.nullProcessor,
-    getNotificationMessage: selectorMap =>
-        isEmpty(selectorMap)
-            ? 'Congratulations!\n\nAutomated checks found no issues on this page.'
-            : null,
+    getNotificationMessage: (selectorMap, key, warnings) => {
+        const iframeCoda =
+            '\nThere are iframes in the target page. Use FastPass or Assessment to provide additional permissions.';
```

Comment:
This assumes that all warnings will be iframe warnings: are we sure we want to make that assumption (we are not doing so as far as I am aware elsewhere)?

Reply:
Make sense. Updated to explicitly check for the proper id. I also refactor the code to reduce the amount of nested if's.